aapl_IEX_model.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
<p>return new DF, dont change in place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>do in in place - better memory usage</p>